metadata-models/docs/entities/dataPlatform.md

```diff
@@ -3,9 +3,3 @@
 Data Platforms are systems or tools that contain Datasets, Dashboards, Charts, and all other kinds of data assets modeled in the metadata graph.
 
 Examples of data platforms are `redshift`, `hive`, `bigquery`, `looker`, `tableau` etc.
-
-## Identity
-
-Data Platforms are identified by the name of the technology. A complete list of currently supported data platforms is available [here](https://raw.githubusercontent.com/datahub-project/datahub/master/metadata-service/restli-servlet-impl/src/main/resources/DataPlatformInfo.json).
```

Comment:
the file we want here is `metadata-service/war/src/main/resources/boot/data_platforms.json`